server/api/routes/groupsRoute.js

```diff
@@ -0,0 +1,92 @@
+const router = require('express').Router();
+const {
+  Group,
+  Group_Post,
+  Group_Member,
+  Group_Post_Like,
+  User,
+} = require('../../db');
+
+// get all groups
+router.get('/', async (req, res, next) => {
+  try {
+    const allGroups = await Group.findAll();
+    res.status(200).json(allGroups);
+  } catch (e) {
+    console.log('Backend issue fetching all groups');
+    next(e);
+  }
+});
+
+// get all groups per member
+
```

Comment:
NO AUTH REQUIREMENTS

axios.get('api/groupsGuest')
'/groupsGuest/' = all users who care all groups

// REQUIRE LOGGED IN aka requireToken auth:
/api/user/1/groups/groupsCreated/1/ -> edit, delete
/api/user/1/groups/groupMembership/2/ -> delete myself from group


axios.get('/api/users/1/groups/member') MERGE
'/groups/:id'
/users/id/groups/....
'/admin'
'/membership' = loggedin use seeing what groups they're a member of
'/createdgroups' = loggedin see what groups they created

Reply:
(for reference)